snowexsql/tables/measurement_type.py

```diff
@@ -8,10 +8,12 @@ class MeasurementType(Base):
     """
     Table to store the measurement types
     """
-    __tablename__ = 'measurements'
+    __tablename__ = 'measurement_type'
 
-    name = Column(String())
+    name = Column(Text)
     units = Column(Text)
+    derived = Column(Boolean, default=False)
+
 
 
 class HasMeasurement:
```

Comment:
For consistency I think we should also rename this class to `HasMeasurementType`